packages/packages/api/src/medical/models/common/address.ts

```diff
@@ -2,12 +2,12 @@ import { z } from "zod";
 import { usStateSchema } from "./us-data";
 
 export const addressSchema = z.object({
-  addressLine1: z.string().min(1),
+  addressLine1: z.string().optional(),
   addressLine2: z.string().optional().nullable(),
-  city: z.string().min(1),
-  state: usStateSchema,
-  zip: z.string().length(5),
-  country: z.string().min(1),
+  city: z.string().optional(),
+  state: usStateSchema.optional(),
+  zip: z.string(),
+  country: z.string().optional(),
 });
 
```

Comment:
If this looks fine ill have to deploy a new version and add commit to the PR

Reply:
I had made more of the address schemas optional but realized the only one really affecting this is this one as the other ones are our internal schemas